lib/pharos/terraform/base_command.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'open3'
+
+module Pharos::Terraform
+  class BaseCommand < Pharos::Command
+    options :yes?
+
+    option "--workspace", "NAME", "terraform workspace", default: "default"
+
+    def tf_workspace
+      unless run_cmd("terraform workspace select #{workspace} 2> /dev/null").zero?
```

Comment:
Workspaces make multi-cluster setups easier to handle (same config for all clusters).